server/chat-plugins/suspect-tests.ts

```diff
@@ -0,0 +1,101 @@
+import {FS} from '../../lib/fs';
+
+const SUSPECTS_FILE = 'config/chat-plugins/suspects.json';
+
+interface SuspectTest {
+	tier: string;
+	suspect: string;
+	date: string;
+	url: string;
+}
+
+export const suspectTests: {[format: string]: SuspectTest} = JSON.parse(FS(SUSPECTS_FILE).readIfExistsSync() || "{}");
+
+function saveSuspectTests() {
+	FS(SUSPECTS_FILE).writeUpdate(() => JSON.stringify(suspectTests));
+}
+
+export const commands: ChatCommands = {
+	suspect: 'suspects',
+	suspects: {
+		''(target, room, user) {
+			if (!Object.keys(suspectTests).length) {
+				return this.errorReply("There are no suspect tests running.");
+			}
+			if (!this.runBroadcast()) return;
+
+			let buffer = '<strong>Suspect tests currently running:</strong>';
+			for (const i of Object.keys(suspectTests)) {
+				buffer += '<br />';
+				buffer += `${suspectTests[i].tier}: <a href="${suspectTests[i].url}">${suspectTests[i].suspect}</a> (${suspectTests[i].date})`;
+			}
+			return this.sendReplyBox(buffer);
+		},
+
+		edit: 'add',
+		add(target, room, user) {
+			this.checkCan('gdeclare');
+
+			const [tier, suspect, date, url] = target.split(',');
+			if (!(tier && suspect && date && url)) {
+				return this.parse('/help suspects');
+			}
+
+			const format = Dex.getFormat(tier);
+			if (!format.exists) return this.errorReply(`"${tier}" is not a valid tier.`);
+
+			const suspectString = suspect.trim();
+
+			const [month, day] = date.trim().split(date.includes('-') ? '-' : '/');
+			const isValidDate = /[0-1]?[0-9]/.test(month) && /[0-3]?[0-9]/.test(day);
+			if (!isValidDate) return this.errorReply("Dates must be in the format MM/DD.");
+			const dateActual = `${month}/${day}`;
+
+			const urlActual = url.trim();
+			if (
+				!['https://www.smogon.com/forums/threads/', 'https://www.smogon.com/forums/posts/']
+					.some(prefix => urlActual.startsWith(prefix))
+			) {
+				return this.errorReply("Suspect test URLs must be Smogon threads or posts.");
+			}
+
+			this.privateGlobalModAction(`${user.name} ${suspectTests[format.id] ? "edited the" : "added a"} ${format.name} suspect test.`);
+			this.globalModlog('SUSPECTTEST', null, `by ${user.id}: ${suspectTests[format.id] ? "edited" : "added"} ${format.name}`);
```

Comment:
```suggestion
			this.globalModlog('SUSPECTTEST ${suspectTests[format.id] ? "EDIT" : "ADD"}', null, `by ${user.id}: ${format.name}`);
```

or ADDSUSPECTTEST, something of the sort; doing `SUSPECTTEST [action]` lets doing `/ml SUSPECTTEST` show all modlog entries related to it, so its more of a QoL matter ig

Reply:
This is what it shows currently.

<img width="523" alt="image" src="https://user-images.githubusercontent.com/1627001/94367784-d2483d00-00f1-11eb-80e4-1e970fa8dd04.png">

Seems fine? I think it's more readable than your suggestion.